Rework update and rollback docs (#191)
flatcar-linux/flatcar-docs: docs/setup/debug/manual-rollbacks.md

**Note:** Neither performing a manual rollback nor forcing a channel downgrade are recommended.

## Automated rollbacks

The rollback to the previously installed version is done by GRUB and happens automatically if `update-engine` had no chance to mark the version as successful.

> This marking happens when the new version is booted and keeps running for around two minues, at which point `update-engine` will mark the version as successful (how this works in detail is explained below).

Reviewer: ```suggestion
This marking happens when the new version is booted and keeps running for around two minutes, at which point `update-engine` will mark the version as successful (how this works in detail is explained below).
```

> To extend the automatic rollback logic to cover your important systemd services, you could make them a requirement for the `update-engine.service`.

Reviewer: ```suggestion
To extend the automatic rollback logic to cover your important systemd services, you could make them as requirement for the `update-engine.service`.
```

Note that `update-engine` will still try to update which can cause a loop with disruptions due to the reboots.
You can disable automatic updates by setting `SERVER=disabled` in `/etc/flatcar/update.conf`.

## Rollback with `flatcar-update`

> While you can roll back to the previously installed version manually with the rest of this guide, you can also install any version to the inactive partition with the `flatcar-update` tool.

Reviewer: ```suggestion
While you can rollback to the previously installed version manually with the rest of this guide, you can also install any version to the inactive partition with the `flatcar-update` tool.
```

To stay consistent with other `rollback`.
